### PR TITLE
Jr bugfix duplicateevents

### LINKED ISSUE #91: BUG - Duplicate domEvents bug where when you sign out and in events are duplicated

### PR DESCRIPTION
**Pull request by jrobinson0529**
## Description
Fixed bug causing domEvents to duplicate upon sign out and sign in.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by jrobinson0529** [comment]
Test thoroughly to make sure all modules are working as intended. I had to change almost all of domEvents!

**Review by hmparsons-sde — APPROVED** [review]
Looks good & works on my end

File: src/javascripts/events/domEvents.js [hunk 1]
```diff
@@ -35,195 +37,234 @@ import showFoodReadOnly from '../components/readOnlyPrinters/showFoodReadOnly';
...
+    e.preventDefault();
+    getEvents().then((x) => {
       const showObject = {
         name: document.querySelector('#showName').value,
         image: document.querySelector('#image').value,
         date: document.querySelector('#date').value,
         description: document.querySelector('#description').value,
       };
-      updateShow(firebaseKey, showObject, id).then((showsArray) => printShows(showsArray));
+      updateShow(firebaseKey, showObject, x.preventDefault).then((showsArray) => printShows(showsArray));
```
Comment (chiestroud) [inline src/javascripts/events/domEvents.js]: @jrobinson0529 is this the one causing the issue? x.firebaseKey should solve???

Comment (jrobinson0529) [reply src/javascripts/events/domEvents.js]: Fixed!

**Review by hmparsons-sde — APPROVED** [review]

**Review by katherinevfry — APPROVED** [review]
This all looks great, Jesse!

**Pull request merged**